scripts/_helpers.js

```diff
@@ -0,0 +1,36 @@
+var R = require('ramda');
+var head = R.head;
+var assoc = R.assoc;
+var rx = /\D/g;
+var outDir = './tmp';
+var opts = {
+  url: 'https://api.github.com/repos/ramda/ramda/tags',
+  headers: {'user-agent': 'node.js'}
+};
+
+function byName(a, b) { 
+  return versionToNum(b.name) - versionToNum(a.name);
+}
+
+function fetchOpts(tags) {
+  var tag = head(JSON.parse(tags).sort(byName));
+  return assoc('url', tag.tarball_url, opts)
+}
+
+// [String] -> [String]
+var zeroPad = R.map(function pad(n) { return n.length === 3 ? n : pad('0' + n); });
```

Comment:
`zeroPad` should have type `String -> String`. You could do the mapping in `versionToNum`.